components/ChapterOpeningOtherUser.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <section class="chapter-opening">
+    <div class="wrapper">
+      <h1>{{ title }}</h1>
+      <p class="pt2 pb4 measure-wide">{{ paragraph }}</p>
+      <div class="profile-container">
```

Comment:
besides the other user profile, this component is basically a regular `ChapterOpening`. The current implementation leads to a lot of duplicate styles and logic, which is difficult to maintain and build upon.
It would be more elegant to implement this as part of a regular `ChapterOpening`, and using a slot or a `prop` to pass in an optional component which renders the data in `otherUserProfile`.

Reply:
Thank you for the ideas! however can we do this later? I would prefer to work on the standing issues than redo the entire component architecture

Reply:
I created a new issue for this @ #80

Reply:
Ok, I hear you. Let's merge and keep the refactor for later.